contract-testing/src/impls.rs

```diff
@@ -2058,6 +2060,12 @@ pub fn module_load_v1(module_path: impl AsRef<Path>) -> Result<WasmModule, Modul
     Ok(module)
 }
 
+pub fn module_load_contract() -> Result<WasmModule, ContractModuleLoadError> {
```

Comment:
**Suggestion:** Please add documentation for what the function does and when to use this.